server/src/fty-srr-cmd.cc

```diff
@@ -66,9 +66,11 @@ std::ostream &operator<< (std::ostream &os, const std::vector<T> &vec)
     os << *it;
     return os;
 }
-
+// Utils
 dto::UserData sendRequest (const std::string &action,
                            const dto::UserData &userData);
+auto buildReauthToken(const std::string& sessionToken, const std::string& reauthPasswd) -> std::string;
```

Comment:
Please keep coding still as the other function in this file.

Reply:
With same ident?

Reply:
Done

Reply:
std::string buildReauthToken(const std::string& sessionToken, const std::string& reauthPasswd);

Reply:
If you want do write it, don't hesitate